src/onboarding/Welcome/Welcome.js

```diff
@@ -38,71 +39,67 @@ const Welcome = React.memo(function Welcome({
     [selectorNetworksSorted]
   )
 
+  const suggestedOrgs = useSuggestedOrgs()
+
+  const primaryContent = openMode ? (
+    <OpenOrg onBack={onBack} onOpenOrg={onOpenOrg} />
+  ) : (
+    <div>
+      <DropDown
+        items={selectorNetworksSorted.map(network => network.name)}
+        placeholder={selectorNetworksSorted[0].name}
+        onChange={changeNetwork}
+        wide
+      />
+      <WelcomeAction
+        title="Create an organization"
+        subtitle={<CreateSubtitle error={createError} />}
+        illustration={actionCreate}
+        onActivate={onCreate}
+        hasError={createError[0] !== null && createError[0] !== 'no-account'}
+      />
+      <WelcomeAction
+        title="Open an existing organization"
+        illustration={actionOpen}
+        onActivate={onOpen}
+      />
+    </div>
+  )
+
   return (
-    <div
-      css={`
-        display: grid;
-        align-items: center;
-        justify-content: center;
-      `}
+    <Layout
+      breakpoints={{
+        medium: 84 * GU,
+        large: 112 * GU,
+      }}
     >
-      <Layout
-        breakpoints={{
-          medium: 84 * GU,
-          large: 112 * GU,
-        }}
-      >
-        <Header
-          title="Welcome to Aragon"
-          subtitle="Create your own organization in a few minutes!"
-        />
+      <Header
+        title="Welcome to Aragon"
+        subtitle="Create your own organization in a few minutes!"
+      />
 
+      {suggestedOrgs.length > 0 ? (
```

Comment:
Not sure I understand this change; shouldn't we only not show `<Suggestions>` if `suggestedOrgs.length === 0`?